### PR TITLE
feat(Peer-Group): Add teacher endpoint to add member to peer group

### LINKED ISSUE #75: feat(Peer-Group): New teacher endpoint for adding member to PeerGroup
### Description

Create a new endpoint for teachers to add a member to a PeerGroup at ```/api/peer-group/membership/add/{peerGroupId}/{workgroupId}```

If the workgroup is in another PeerGroup for the same PeerGroupActivity, remove it from the old PeerGroup before placing it in the new PeerGroup.

### PR DESCRIPTION
**Pull request by hirokiterashima**
## Changes
- Add a new endpoint for teachers to add a member to a PeerGroup at ```/api/peer-group/membership/add/{peerGroupId}/{workgroupId}```

## Test
- Approve #73 first
- The endpoint should throw AccessDeniedException if user does not have write permission on the run
- If the workgroup is in another PeerGroup for the same PeerGroupActivity, the workgroup should be removed from the old PeerGroup before being place in the new PeerGroup.
- If the workgroup is already in the specified PeerGroup, no changes will be made.
- It should return the modified PeerGroup in the response

Closes #75

### CODE REVIEW

File: src/main/java/org/wise/portal/presentation/web/controllers/peergroup/PeerGroupMembershipController.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  private RunService runService;
+
+  @PostMapping("/add/{peerGroupId}/{workgroupId}")
+  PeerGroup addMember(@PathVariable("peerGroupId") PeerGroupImpl peerGroup,
+      @PathVariable("workgroupId") WorkgroupImpl workgroup, Authentication auth) {
+    if (runService.hasWritePermission(auth, peerGroup.getPeerGroupActivity().getRun())) {
+      return peerGroupMembershipService.addMember(peerGroup, workgroup);
+    }
+    throw new AccessDeniedException("Not permitted");
+  }
```
Comment (geoffreykwan) [inline src/main/java/org/wise/portal/presentation/web/controllers/peergroup/PeerGroupMembershipController.java]: I was able to add a workgroup to a Peer Group where the workgroup was not in the run that is associated with the Peer Group.

Should we check that the workgroup is in the run that the Peer Group is associated with?

**Review by geoffreykwan — CHANGES_REQUESTED** [review]
Main functionality works well.

File: src/main/java/org/wise/portal/presentation/web/controllers/peergroup/PeerGroupMembershipController.java (hunk 1 above)
Comment (hirokiterashima) [reply src/main/java/org/wise/portal/presentation/web/controllers/peergroup/PeerGroupMembershipController.java]: Good catch. I made the change.

**Review by geoffreykwan — APPROVED** [review]
Looks good.

**Pull request merged**